Mathlib/Data/Fintype/Basic.lean

```diff
@@ -1187,13 +1187,27 @@ end Trunc
 
 namespace Multiset
 
-variable [Fintype α] [DecidableEq α]
+variable [Fintype α] [DecidableEq α] [Fintype β] [DecidableEq β]
```

Comment:
It looks like you don't need this
```suggestion
variable [Fintype α] [DecidableEq α] [Fintype β]
```
(you might need to use `classical` in the proof)